sklearn/multioutput.py

```diff
@@ -1243,5 +1253,9 @@ def get_metadata_routing(self):
         )
         return router
 
-    def _more_tags(self):
-        return {"multioutput_only": True}
+    def __sklearn_tags__(self):
+        tags = super().__sklearn_tags__()
+        # FIXME
```

Comment:
Maybe not required as we aren't skip the tests here? Or has this got something to do with multi_output and single_output?
```suggestion
```